variants/frontend-react/app/frontend/components/HelloWorld.jsx

```diff
@@ -0,0 +1,13 @@
+import * as PropTypes from 'prop-types';
+import React from 'react';
+
+class HelloWorld extends React.Component {
```

Comment:
These days it tends to be nicer to use functional components powered by [hooks](https://reactjs.org/docs/hooks-intro.html) rather than class components, so I think it would make sense for this to be functional rather than class to encourage that.

It'd look something like:

```
import * as PropTypes from 'prop-types';
import React from 'react';

const HelloWorld = ({ greeting }) => <>Greeting: {greeting}</>;

HelloWorld.propTypes = {
  greeting: PropTypes.string.isRequired
};

export default HelloWorld;
```

Reply:
Thank you. I am going to change it